### DISCUSSION TITLE
Add custom rules to @font-face style

### DISCUSSION THREAD

**Discussion by jago86** [spatie/laravel-google-fonts#15] [General]
Hi everyone. I'm currently improving the score of my webpage in [Google PageSpeed Insights](https://developers.google.com/speed/pagespeed/insights/) and I'm realized that one of the recomendations is to include *font-display: swap;* inside the @font-face style. You can see the recomendation [here](https://web.dev/font-display/?utm_source=lighthouse&utm_medium=unknown#how-to-avoid-showing-invisible-text). There's any way to include this rule in the css?

**Comment by fecony** [comment]
Hey @jago86!

According to Google Fonts [docs](https://developers.google.com/fonts/docs/getting_started#use_font-display) you can add `&display=swap` to url and it will add `font-display: swap;`.

Example: https://fonts.googleapis.com/css2?family=Inter:ital,wght@0,400;0,700;1,400;1,700&display=swap
